src/test/java/org/folio/edge/controller/InnReachProxyControllerTest.java

```diff
@@ -107,18 +110,18 @@ void shouldUpdateRequestHeaders() {
     SecureStore secureStore = SecureStoreFactory.getSecureStore(AwsParamStore.TYPE, PropertiesUtil.getProperties(null));
     SecureStore ephermalSecureStore = SecureStoreFactory.getSecureStore(EphemeralStore.TYPE, PropertiesUtil.getProperties(null));
     Optional<String> tenatMapping1 = null;
-   Optional<String> tenatMapping = SecureTenantsProducer.getTenants(PropertiesUtil.getProperties(null), secureStore,
+
+    Optional<String> tenatMapping = SecureTenantsProducer.getTenants(PropertiesUtil.getProperties(null), secureStore,
       "6b583dfe-8c34-40bb-a520-5b49b23edb3d:diku");
 
-   try {
-     tenatMapping1 = SecureTenantsProducer.getTenants(PropertiesUtil.getProperties(null), ephermalSecureStore,
-       "6b583dfe-8c34-40bb-a520-5b49b23edb3d:diku");
-   }
-   catch (Exception e){
-     assertNull(tenatMapping1);
-   }
+    try {
+      tenatMapping1 = SecureTenantsProducer.getTenants(PropertiesUtil.getProperties(null), ephermalSecureStore,
+        "6b583dfe-8c34-40bb-a520-5b49b23edb3d:diku");
+    } catch (Exception e) {
+      assertNull(tenatMapping1);
+    }
 
-   assertTrue(tenatMapping.isEmpty());
+    assertTrue(tenatMapping.isEmpty());
```

Comment:
This test is failing locally. Please take a look at tests from edge-api-utils: 
https://github.com/folio-org/edge-api-utils/blob/7eb15d1fec569de662ef87e295c4b548eb1736a8/src/test/java/org/folio/edge/api/utils/security/SecureStoreFactoryTest.java#L35

https://github.com/folio-org/edge-api-utils/blob/master/src/test/java/org/folio/edge/api/utils/security/AwsParamStoreTest.java

Reply:
Moreover, I don't think these checks belong to this test method (which is intended to just verify that request headers are updated). If you want to test Secure store related logic, then it make sense to create a dedicated test class or classes for this purpose.

Reply:
Thanks. I have fixed it based on your suggestion.